RELEASE-NOTES.txt

```diff
@@ -1,3 +1,10 @@
+1.10.0
+------
+* Adding a block from the post title now shows the add block here indicator.
```

Comment:
There is one final edge case for this (adding from the post title on a new post when there is just the placeholder text and no paragraph block yet) is still not fixed. Should we pull this from the release notes until that is fixed?

Reply:
I think this is fine as it's included in this release

Reply:
Aww, but that screws up my plan of changing *all* the release notes. 😉